scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -358,30 +366,31 @@ class FossId internal constructor(
         revision: String,
         projectCode: String,
         projectName: String
-    ): String {
+    ): Pair<String, String> {
         val existingScan = scans.recentScansForRepository(url, revision).findLatestPendingOrFinishedScan()
 
-        val scanCode = if (existingScan == null) {
+        val scanCodeAndIdPair = if (existingScan == null) {
```

Comment:
It's probably fine to drop "Pair" from the name.

Reply:
The name is still the same (in this commit). Did you intend to change it?

Reply:
@sschuberth I have moved the changes to the respective commit.